lib/package.json

```diff
@@ -1,11 +1,8 @@
 {
   "name": "@ledgerhq/exchange-sdk",
-  "version": "0.0.2-0",
+  "version": "0.0.0",
```

Comment:
For now we will no longer rely on this version and instead pull from npm registry during the workflow.